plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/proceed-to-checkout-block/block.tsx

```diff
@@ -103,15 +103,18 @@ const Block = ( {
 		[]
 	);
 
+	// Get the zoom level to determine if the sticky container should be displayed.
+	const zoomLevel = ( window.outerWidth / window.innerWidth ) * 100;
+
 	return (
 		<div className={ clsx( 'wc-block-cart__submit', className ) }>
 			{ positionReferenceElement }
 			{ /* The non-sticky container must always be visible because it gives height to its parent, which is required to calculate when it becomes visible in the viewport. */ }
 			<div className="wc-block-cart__submit-container">
 				{ submitContainerContents }
 			</div>
-			{ /* If the positionReferenceElement is below the viewport, display the sticky container. */ }
-			{ positionRelativeToViewport === 'below' && (
+			{ /* If the positionReferenceElement is below the viewport and if the the zoom level is <= 100%, display the sticky container. */ }
```

Comment:
Since we're just adding a new class and style based on the conditions, I think we can simplify this code by using conditional classes with clsx.

Example: 

```js

	const zoomLevel = ( window.outerWidth / window.innerWidth ) * 100;
	const isStickyContainer =
		positionRelativeToViewport === 'below' && zoomLevel <= 100;

	const submitContainerClass = clsx( 'wc-block-cart__submit-container', {
		'wc-block-cart__submit-container--sticky': isStickyContainer,
	} );

	return (
		<div className={ clsx( 'wc-block-cart__submit', className ) }>
			{ positionReferenceElement }
			<div
				className={ submitContainerClass }
				style={ isStickyContainer ? { backgroundColor } : undefined }
			>
				{ submitContainerContents }
			</div>
		</div>
	);

```

Reply:
Thanks for your suggestion, @tarunvijwani. As I've opted for a different approach, based on our discussion in p1718875553124459-slack-C8X6Q7XQU, I'd prefer to leave the logic as it is for now to prevent accidentally introducing a regression.